src/dmd/mars.d

```diff
@@ -2076,7 +2076,6 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
                             params.dtorFields = FeatureState.enabled;
                             break;
                         case 14_488:
-                            params.vcomplex = true;
```

Comment:
Maybe some sort of heads-up should be given here to inform that this is the default now?

Reply:
I'd rather have this removed, but @Geod24 had other ideas.  My assumption is that this compiler switch is not known by many and everyone is really using `-transition=complex` (if they care about it).